massviewpoint/features/org.polarsys.capella.vp.mass.feature/feature.xml

```diff
@@ -68,4 +68,11 @@
          version="0.0.0"
          unpack="false"/>
 
```

Comment:
can you do global pass to EPL-2 rather than EPL-1

Reply:
Done (with a lot of files impacted)